chapters/appendix-IV-SPDX-license-expressions.md

```diff
@@ -1,175 +1,174 @@
+The operators described above have the following precedence, from highest (binding tightest) at the top, to lowest (loosest) at the bottom:
 
+    Grouping
     +
     WITH
     AND
     OR
 
-where a lower order operator is applied before a higher order operator.
-
 For example, the following expression:
 
-    LGPL-2.1 OR BSD-3-Clause AND MIT 
+    LGPL-2.1 OR BSD-3-Clause AND MIT
 
-represents a license choice between either LGPL-2.1 and the expression BSD-3-Clause AND MIT because the AND operator takes precedence over (is applied before) the OR operator.
+represents a license choice between either `LGPL-2.1` and the expression `BSD-3-Clause AND MIT` because the `AND` operator takes precedence over (is applied before) the `OR` operator.
 
-When required to express an order of precedence that is different from the default order a \<license-expression> can be encapsulated in pairs of parentheses: ( ), to indicate that the operators found inside the parentheses takes precedence over operators outside. This is also similar to the use of parentheses in an algebraic expression e.g., (5+7)/2.
+## IV.3 License Expressions in XML <a name="IV.3"></a>
 
-For instance, the following expression:
+### IV.3.1 The `+` Operator <a name="IV.3.1"></a>
+
+FIXME: How are you supposed to cover this in XML?
 
-    (MIT AND (LGPL-2.1+ OR BSD-3-Clause))
+### IV.3.2 Exception `WITH` Operator <a name="IV.3.2"></a>
 
-states the OR operator should be applied before the AND operator. That is, one should first select between the LGPL-2.1+ or the BSD-3-Clause license before applying the MIT license.
+A License Exception can be expressed in XML via a `<spdx:LicenseException>` element.  This element has the following children:
 
-### 5) License Expressions in RDF <a name="rdf-expr"></a>
+* [`rdfs:comment`][rdfs:comment] element describing the nature of the exception.
+* [`rdfs:seeAlso`][rdfs:seeAlso] element (optional) referencing external sources of information on the exception.
+* `spdx:example`: Text describing examples of this exception.
+* `spdx:name`: The full human readable name of the item.
+* `spdx:licenseExceptionId`: The identifier of an exception in the SPDX License List to which the exception applies.
 
-A conjunctive license can be expressed in RDF via a `<spdx:ConjunctiveLicenseSet>` element, with an spdx:member property for each element in the conjunctive license. Two or more members are required.
+    FIXME: I don't think this element makes sense.  What you want is a way to represent a `simple-expression` based on `WITH`.  This subsection seems to be focused on defining exceptions, but it does not cover *consuming* those license exceptions.  And “and exception … to which the exception applies” is really not clear; is it the left or right `WITH` operand?  What happens for licenses/exceptions that are not in the SPDX License List?
```

Comment:
There is another SPDX class that needs to be described.  The With operator:

The WITH operator can be expressed in XML via a `<spdx:WithExceptionOperator>` element, with an `spdx:member` property for the license and an `spdx:licenseException` property for the exception.

     <spdx:WithExceptionOperator>
          <spdx:member rdf:resource="http://spdx.org/licenses/GPL-2.0"/>
          <spdx:licenseException rdf:resource="https://spdx.org/licenses/FLTK-exception">
      </spdx:WithExceptionOperator>

Reply:
> The WITH operator can be expressed in XML via a `<spdx:WithExceptionOperator>` element…

That makes sense to me.  But I expect we want to drop the `<spdx:LicenseException>` element, because as far as I can tell we provide no other way for authors to define exceptions.  They can only use exceptions from [our list][1], or they can [define a `LicenseID-*` that includes both the license and the exception][2].  But I see nowhere outside of this appendix that suggests they can define their own exceptions.

[1]: https://github.com/spdx/spdx-spec/blob/cfa1b9d08903befdf03e669da6472707b7b60cb9/chapters/appendix-I-SPDX-license-list.md#i2-exceptions-list-
[2]: https://github.com/spdx/spdx-spec/blob/cfa1b9d08903befdf03e669da6472707b7b60cb9/chapters/6-other-licensing-information-detected.md#61-license-identifier-

Reply:
The reference for the ```<spdx:LicenseException>``` is still of value for those wanting to parse the details out of the exception.  We may want to move this to a different section, however, so we don't confuse the reader as to whether they can define license exceptions.  Feel free to remove it for now and we can add it back in when we document the other fields.